hubblestack/extmods/fdg/osquery.py

```diff
@@ -51,6 +51,7 @@ def query(query_sql, osquery_args=None, osquery_path=None, format_chained=True,
         osquery_args = []
     return _osquery(query_sql, args=osquery_args, osquery_path=osquery_path)
 
+    return _osquery(query_sql, args=osquery_args, osquery_path=osquery_path)
```

Comment:
Seems like a duplicate return statement. @jettero  Can you verify this ?

Reply:
Fixed, thanks. Fallout from the very complicated backport. The forward port is going to be just as bad...